### PR TITLE
Feature/update wrapper ros2

### PR DESCRIPTION
**Pull request by adev4a**
# PR Details
This PR adds updates the velodyne novatel wrapper to ros2.
## Description

ros1 wrapper updated to ros2

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
[CARMA Contributing Guide](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) 
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by msmcconnell** [comment]
I have retargeted the PR at foxy/develop

File: docker/install.sh [hunk 1]
```diff
@@ -14,6 +14,17 @@
...
 
-source /opt/autoware.ai/ros/install/setup.bash
-cd ~/
-colcon build --cmake-args -DCMAKE_BUILD_TYPE=Release
+# Source ros2
+source /opt/ros/foxy/setup.bash
+# Get driver
+sudo apt-get update && sudo apt install curl gnupg2 lsb-release
+sudo curl -sSL https://raw.githubusercontent.com/ros/rosdistro/master/ros.key  -o /usr/share/keyrings/ros-archive-keyring.gpg
+echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/ros-archive-keyring.gpg] http://packages.ros.org/ros2/ubuntu $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/ros2.list > /dev/null
```
Comment (msmcconnell) [inline docker/install.sh:22]: Is this ros2 key not already in carma-base?

File: velodyne_lidar_driver_wrapper/launch/velodyne_lidar_driver_wrapper_launch.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+from ament_index_python import get_package_share_directory
+from launch import LaunchDescription
+import launch
+from launch_ros.actions import ComposableNodeContainer
+from launch_ros.descriptions import ComposableNode
+from launch.actions import DeclareLaunchArgument
+from launch.substitutions import LaunchConfiguration
+
+from launch.actions import IncludeLaunchDescription
+from launch.launch_description_sources import PythonLaunchDescriptionSource
```
Comment (msmcconnell) [inline velodyne_lidar_driver_wrapper/launch/velodyne_lidar_driver_wrapper_launch.py:24]: Add 
```from carma_ros2_utils.launch.get_current_namespace import GetCurrentNamespace```

File: velodyne_lidar_driver_wrapper/launch/velodyne_lidar_driver_wrapper_launch.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+    #  Get parameter file path
+    param_file_path = os.path.join(
+        get_package_share_directory('velodyne_lidar_driver_wrapper'), 'config/parameters.yaml')
+
+    # Launch node(s) in a carma container to allow logging to be configured
+    velodyne_lidar_wrapper_container = ComposableNodeContainer(
+        package='carma_ros2_utils',
+        name='velodyne_lidar_driver_wrapper_container',
+        namespace='/',
```
Comment (msmcconnell) [inline velodyne_lidar_driver_wrapper/launch/velodyne_lidar_driver_wrapper_launch.py:50]: Use GetCurrentNamespace() here

File: velodyne_lidar_driver_wrapper/launch/velodyne_lidar_driver_wrapper_launch.py [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+        executable='carma_component_container_mt',
+        composable_node_descriptions=[
+            
+            # Launch the core node(s)
+            ComposableNode(
+                    package='velodyne_lidar_driver_wrapper',
+                    plugin='velodyne_lidar_driver_wrapper::Node',
+                    name='velodyne_lidar_driver_wrapper_node',
+                    namespace="/",
```
Comment (msmcconnell) [inline velodyne_lidar_driver_wrapper/launch/velodyne_lidar_driver_wrapper_launch.py:59]: use GetCurrentNamespace() here

**Review by msmcconnell — CHANGES_REQUESTED** [review]

File: docker/install.sh (hunk 1 above)
Comment (adev4a) [reply docker/install.sh:22]: I see this key being added in carma base https://github.com/usdot-fhwa-stol/carma-base/blob/a9f0aa253ba9baa825f04bcc9e6354bccf5249bb/Dockerfile#L63
But it seems slightly different

**Review by msmcconnell — APPROVED** [review]

**Review by msmcconnell — APPROVED** [review]

**Pull request merged**